packages/perseus/src/index.ts

```diff
@@ -125,6 +125,9 @@ export {
     getQuadraticCoords,
     getAngleCoords,
 } from "./widgets/interactive-graphs/reducer/initialize-graph-state";
+// This export is to support necessary functionality in the perseus-editor package.
+// It should be removed if widgets and editors become colocated.
+export {getClockwiseAngle} from "./widgets/interactive-graphs/math";
 
```

Comment:
After reviewing kmath and some other possible locations, it seemed best to simply export this. I feel like this is a relatively safe function to export and is using the same approach as the getAngleCoords. 

I'm happy to take any feedback on this however!

Reply:
Question: What would the downsides of exporting it be?